README.md

```diff
@@ -1,49 +1,30 @@
-## contexture-react
-`contexture-react` is a suite of react components for building contexture interfaces
+# contexture-react
 
+React components for building contexture interfaces https://smartprocure.github.io/contexture-react
 
 [![CircleCI](https://circleci.com/gh/smartprocure/contexture-react.svg?style=svg)](https://circleci.com/gh/smartprocure/contexture-react)
 [![Greenkeeper badge](https://badges.greenkeeper.io/smartprocure/contexture-react.svg)](https://greenkeeper.io/)
 [![npm version](https://badge.fury.io/js/contexture-react.svg)](https://badge.fury.io/js/contexture-react)
 ![dependencies](https://david-dm.org/smartprocure/contexture-react.svg)
 [![Storybook](https://img.shields.io/badge/BUILT_WITH-STORYBOOK-f1618c.svg?style=for-the-badge)](https://smartprocure.github.io/contexture-react)
 
-
 # Documentation
 
-Documentation is largely provided via our [Storybook](https://smartprocure.github.io/contexture-react).
+Documentation lives under `./docs` and our [Storybook](https://smartprocure.github.io/contexture-react)**.
 
 **The most complete example is the [Live Demo using IMDB data and the Grey Vest theme](https://smartprocure.github.io/contexture-react/?path=/story/live-demos-imdb--grey-vest-theme)**
```

Comment:
this path will be gone when v2 goes live, so we should probably change it now. The post-refactor equivalent to the grey vest demo would be the SearchLayout demo thing ([.../story/live-demos-imdb-search--dynamic-search-layout]()), if we wanna continue linking to a specific one.

Also, we have multiple actually-useful live demos now, so we could change the wording a bit. Maybe something like 

> The most complete usage examples are our live demos, particularly the [themeable SearchLayout demo](https://smartprocure.github.io/contexture-react/?path=/story/live-demos-imdb--grey-vest-theme).

Reply:
Or maybe it's irrelevant now, since our storybook has gotten so much better.

Reply:
Up to you! I'm fine with it either way - I think the [docs, components, demo] ordering is sensible, but it'd be neat to give the demos more visibility. Having the searchLayout demo be the first story users see (vs. seeing the readme a second time) would probably be worth shuffling things around.

If there's a way to have storybook redirect to a specific story instead of the first one, that would be an option too - I didn't see anything in the docs about that, though, so I'm not sure storybook supports it.

Reply:
Prefixing with 01, 02, 03, etc... will do the job _if_ we want to have custom orderings.